pages/LandingPage.module.css

```
@@ -1,5 +1,6 @@
 .UserGuide-article {
   min-width: 390px;
+  max-width: 60%;
```

https://kes3583.medium.com/vh-vw-91be52024f49

using `vw` is recommended.

```suggestion
  max-width: 60vw;
```